programs/account-compression/src/instructions/register_program.rs

```diff
@@ -17,7 +17,7 @@ pub struct RegisterProgramToGroup<'info> {
     #[account( mut, constraint= authority.key() == group_authority_pda.authority @AccountCompressionErrorCode::InvalidAuthority)]
     pub authority: Signer<'info>,
     /// CHECK: TODO: check that upgrade authority is signer.
```

Comment:
remove todo ?

Reply:
want to keep this todo for a next release